compose/app/build.gradle.kts

```diff
@@ -0,0 +1,93 @@
+plugins {
+    id("com.android.application")
+    id("org.jetbrains.kotlin.android")
+}
+
+android {
+    namespace = "com.chartboost.mediation.sdk.demo"
+    compileSdk = 33
+
+    defaultConfig {
+        applicationId = "com.chartboost.mediation.sdk.demo"
+        minSdk = 21
+        targetSdk = 33
+        versionCode = 1
+        versionName = "1.0"
+
+        testInstrumentationRunner = "androidx.test.runner.AndroidJUnitRunner"
+        vectorDrawables {
+            useSupportLibrary = true
+        }
+    }
+
+    buildTypes {
+        release {
+            isMinifyEnabled = false
+            proguardFiles(
+                getDefaultProguardFile("proguard-android-optimize.txt"),
+                "proguard-rules.pro"
+            )
+        }
+    }
+    compileOptions {
+        sourceCompatibility = JavaVersion.VERSION_1_8
+        targetCompatibility = JavaVersion.VERSION_1_8
+    }
+    kotlinOptions {
+        jvmTarget = "1.8"
+    }
+    buildFeatures {
+        compose = true
+    }
+    composeOptions {
+        kotlinCompilerExtensionVersion = "1.4.3"
+    }
+    packaging {
+        resources {
+            excludes += "/META-INF/{AL2.0,LGPL2.1}"
+        }
+    }
+}
+
+dependencies {
+    // Chartboost Mediation SDK
+    implementation("com.chartboost:chartboost-mediation-sdk:4.5.0")
+
+    // Chartboost Mediation adapters
+    implementation("com.chartboost:chartboost-mediation-adapter-adcolony:+")
+    implementation("com.chartboost:chartboost-mediation-adapter-admob:+")
+    implementation("com.chartboost:chartboost-mediation-adapter-amazon-publisher-services:+")
+    implementation("com.chartboost:chartboost-mediation-adapter-applovin:+")
+    implementation("com.chartboost:chartboost-mediation-adapter-chartboost:+")
+    implementation("com.chartboost:chartboost-mediation-adapter-digital-turbine-exchange:+")
+    implementation("com.chartboost:chartboost-mediation-adapter-google-bidding:+")
+    implementation("com.chartboost:chartboost-mediation-adapter-hyprmx:+")
+    implementation("com.chartboost:chartboost-mediation-adapter-inmobi:+")
+    implementation("com.chartboost:chartboost-mediation-adapter-ironsource:+")
+    implementation("com.chartboost:chartboost-mediation-adapter-meta-audience-network:+")
+    implementation("com.chartboost:chartboost-mediation-adapter-mintegral:+")
+    implementation("com.chartboost:chartboost-mediation-adapter-mobilefuse:+")
+    implementation("com.chartboost:chartboost-mediation-adapter-pangle:+")
+    implementation("com.chartboost:chartboost-mediation-adapter-reference:+")
+    implementation("com.chartboost:chartboost-mediation-adapter-tapjoy:+")
+    implementation("com.chartboost:chartboost-mediation-adapter-unity-ads:4.4.8.0.2")
```

Comment:
why is this one 4.4.8.0.2?

Reply:
haha. I hard-coded that to test something totally unrelated. will revert